pkg/logql/shardmapper.go

```diff
@@ -323,8 +323,8 @@ var shardableOps = map[string]bool{
 	OpTypeCount: true,
 
 	// range vector ops
-	OpTypeCountOverTime: true,
-	OpTypeRate:          true,
+	OpRangeTypeCount: true,
+	OpRangeTypeRate:  true,
```

Comment:
Please add the byte methods here -- they should be shardable :)

Reply:
I was waiting for you input on that, that's all I need ?

Reply:
got it.

Reply:
I checked and currently you'll need to add cases here as well:
```
func (m ShardMapper) mapRangeAggregationExpr(expr *rangeAggregationExpr, r *shardRecorder) SampleExpr {
	switch expr.operation {
	case OpTypeCountOverTime, OpTypeRate:
		// count_over_time(x) -> count_over_time(x, shard=1) ++ count_over_time(x, shard=2)...
		// rate(x) -> rate(x, shard=1) ++ rate(x, shard=2)...
		return m.mapSampleExpr(expr, r)
	default:
		return expr
	}
}
```

However, it feels weird having this logic in two places. I think it makes sense to remove the `mapRangeAggregationExpr` entirely and just defer to `mapSampleExpr`